tests/datapipeline/test_datapipeline.py

```diff
@@ -0,0 +1,101 @@
+"""Verify AWS Data Pipeline Creation."""
+from unittest import mock
+import pytest
+
+import copy
+import awscli.customizations.datapipeline.translator
+from foremast.datapipeline.datapipeline import AWSDataPipeline
+
+GOOD_DEF = 	{
+        "objects": [],
+        "parameters": [],
+        "values": {}
+        }
+
+BAD_DEF = 	{
+        "no_object": [],
+        "parameters": [],
+        "values": {}
+        }
+
+TEST_PROPERTIES = {
+    'test_env': {
+        'datapipeline': {
+            'name': 'Test Pipeline',
+            'description': "this is a test",
+            'json_definition': GOOD_DEF
+        }
+    }
+}
+
+@mock.patch('foremast.datapipeline.datapipeline.boto3.Session.client')
+@mock.patch('foremast.datapipeline.datapipeline.get_details')
+@mock.patch('foremast.datapipeline.datapipeline.get_properties')
+def test_create_datapipeline(mock_get_properties, mock_get_details, mock_boto3):
+    """Check data pipeline creation"""
+    generated = {"project": "test"}
+    properties = copy.deepcopy(TEST_PROPERTIES)
+    mock_get_details.return_value.data = generated
+    mock_get_properties.return_value = properties
+    mock_boto3.return_value.create_pipeline.return_value = {}
+
+    dp = AWSDataPipeline(app='test_app', env='test_env', region='us-east-1', prop_path='other')
+    dp.create_datapipeline()
+    assert dp.pipeline_id == None
+
+@mock.patch('foremast.datapipeline.datapipeline.boto3.Session.client')
+@mock.patch('foremast.datapipeline.datapipeline.get_details')
+@mock.patch('foremast.datapipeline.datapipeline.get_properties')
+def test_good_set_pipeline_definition(mock_get_properties, mock_get_details, mock_boto3):
```

Comment:
yeah, I want to assert that an exception is NOT raised. I couldnt figure out how to do that. The test fails properly if the exception is raised